src/index.js

```diff
@@ -20,15 +22,142 @@ function createElement(tagName, props, ...children) {
   return element;
 }
 
-function render() {
+const display = (left, operator, right = 0, result, currentState = 'left') => {
+  if (currentState === 'left') return left;
+  if (currentState === 'result') return result;
+  if (currentState === 'right') {
+    return right || left;
+  }
+  return 0;
+};
+
+
+const plus = (x, y) => x + y;
+const minus = (x, y) => x - y;
+const multiply = (x, y) => x * y;
+const divide = (x, y) => (x ? x / y : 0);
+
+function calculate(operator) {
+  return (x, y) => {
+    const operators = {
+      '+': plus(x, y),
+      '-': minus(x, y),
+      '*': multiply(x, y),
+      '/': divide(x, y),
+    };
+    return operators[operator];
```

Comment:
이렇게도 처리할 수 있겠네요~ 혹은 operators를 calculate함수에서 추출해도 좋을 것 같아요~
```suggestion
  return (x, y) => {
    const operators = {
      '+': plus(),
      '-': minus(),
      '*': multiply(),
      '/': divide(),
    };
    return operators[operator](x, y);
```